sw-precache.js

```diff
@@ -3,6 +3,7 @@
 var crypto = require('crypto');
 var fs = require('fs');
 var glob = require('glob');
+var lodashTemplateStream = require('lodash-template-stream');
```

Comment:
Use https://github.com/sindresorhus/gulp-template

The above is a duplicate, and a really bad one.